app/Http/Controllers/Three/SignupsController.php

```diff
@@ -102,6 +102,10 @@ public function index(Request $request)
             if (in_array($column, Signup::$indexes)) {
                 $query = $query->orderBy($column, $direction);
             }
+
+            if ($column === 'accepted_quantity') {
+                // maybe do some sorting stuff here
+            }
         }
 
         return $this->paginatedCollection($query, $request);
```

Comment:
Here we pass off the query we have built to `paginatedCollection` and it takes care of the rest. Currently, other orderBy parameters are baked into the query, but since we are trying to order by the sum of the quantity of the accepted posts associated with each signup, it is less straightforward to add to the query (and I haven't figured out a way to get it to work).